src/Tools/ExternalAccess/Xaml/QuickInfo/IXamlQuickInfoService.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Xaml.QuickInfo;
+
+internal interface IXamlQuickInfoService : ILanguageService
+{
+    Task<XamlQuickInfo> GetQuickInfoAsync(TextDocument document, int position, CancellationToken cancellationToken);
```

Comment:
This should be nullable `Task<XamlQuickInfo?>`. Returning empty quick info (when e.g. mouse if over comments) will cause empty tooltip rather than no tooltip.